starport/pkg/cosmosclient/cosmosclient.go

```diff
@@ -29,10 +29,10 @@ import (
 	"github.com/gogo/protobuf/proto"
 	prototypes "github.com/gogo/protobuf/types"
 	"github.com/pkg/errors"
-	rpchttp "github.com/tendermint/tendermint/rpc/client/http"
-
 	"github.com/tendermint/starport/starport/pkg/cosmosaccount"
 	"github.com/tendermint/starport/starport/pkg/cosmosfaucet"
+	"github.com/tendermint/starport/starport/pkg/o"
```

Comment:
I think the 'o' is not perfect name. I would prefer something more descriptive like 'opt' or 'opts'

Reply:
I see the appeal in the name, the `o` it's a package name that can definitely cause strong opinions, in a way it makes sense for it to be short because is going to be used quite frequently and `o.Option` or `o.OptionGroup` are already clear and easy to understand, but the `o` gives a sense of a locally defined variable instead of a package. I don't have a strong opinion on the name, but I have to agree that  `opt.Option` and `opt.OptionGroup` (might be better `opt.Group` in this case) look good too.